azure-storage/pom.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-stream-azure-storage-parent</artifactId>
+	<packaging>pom</packaging>
+
+	<parent>
+		<groupId>org.springframework.cloud.stream.app</groupId>
+		<artifactId>spring-cloud-stream-app-starters</artifactId>
+		<version>1.0.0.BUILD-SNAPSHOT</version>
+	</parent>
+
+	<modules>
+		<module>spring-cloud-starter-stream-sink-wasb</module>
```

Comment:
OK. Thank you for such a quick feedback!
Well, we always can commit into master and release some minimal functionality, but working one.
All other feature we can address in the future or on demand.

So, if you think that the minimal working stuff is already here, just let us know if you are going to address all this our concerns yourself or do you leave it to us to finish it properly and merge finally.

As I said before: it would be much better to move the work to some other top-level branch, but not `master`.